### PR TITLE
Don't open a file handle for a directory in FileIO

### PR DESCRIPTION
**Pull request by Parlane**
Need to test the games that https://code.google.com/p/dolphin-emu/source/detail?r=53d2d9718fe3 fixed:

Poképark Wii 1 & 2
The Beatles: Rock Band save games.
Wii System settings menu

### CODE REVIEW

**Comment by mimimi085181** [comment]
Confirmed to fix Star Wars Force Unleashed II saving problem:
https://code.google.com/p/dolphin-emu/issues/detail?id=6915

Also fixes the home menu in Force Unleashed 2.

**Comment by JMC47** [comment]
Pokepark Wii works fine.  Mimimi showed me the old issue, and I went through the game's intro and it did not hang after a new game.

File: Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_FileIO.cpp [hunk 1]
```diff
@@ -107,7 +107,7 @@ IPCCommandResult CWII_IPC_HLE_Device_FileIO::Open(u32 _CommandAddress, u32 _Mode
 
 	// The file must exist before we can open it
 	// It should be created by ISFS_CreateFile, not here
-	if (File::Exists(m_filepath))
+	if (File::Exists(m_filepath) && !File:: IsDirectory(m_filepath))
```
Comment (Sonicadvance1) [inline Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_FileIO.cpp]: What's with the weird space between File:: and IsDirectory?

Comment (Parlane) [inline Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_FileIO.cpp]: Just trying to be different.

**Comment by skidau** [comment]
The Beatles: Rock Band creates and loads save games fine with this PR.

**Pull request merged**